ote/src/cljs/taxiui/views/front_page.cljs

```diff
@@ -0,0 +1,60 @@
+(ns taxiui.views.front-page
+  "Front page for Taxi UI"
+  (:require [clojure.string :as str]
+            [ote.localization :refer [tr tr-key]]
+            [ote.theme.colors :as colors]
+            [ote.ui.common :refer [linkify]]
+            [ote.app.utils :refer [user-logged-in?]]
+            [re-svg-icons.feather-icons :as feather-icons]
+            [stylefy.core :as stylefy]
+            [taxiui.styles.front-page :as style-front-page]))
+
+(let [host (.-host (.-location js/document))]
+  (def test-env? (or (str/includes? host "test")
+                     (str/includes? host "localhost"))))
+
+(defn test-env-warning []
+  [:div.test-env-warning
+   {:style {:margin "0.2em"
+            :border "4px dashed red"}}
+   [:p {:style {:margin "10px 0px 0px 10px"
+                :font-weight "bold"}}
+    "TÄMÄ ON TESTIPALVELU!!"]
+   [:p {:style {:margin "10px"}}
+    "Julkinen NAP-palvelukatalogi löytyy osoitteesta: "
+    [linkify "https://finap.fi/ote/#/services" "finap.fi"]]
+   [:p {:style {:margin "10px"}}
+    "Lisätietoa NAP-palvelukatalogin taustoista saat osoitteesta "
+    [linkify (tr [:common-texts :footer-livi-url-link])
+     (tr [:common-texts :footer-livi-url-link])]]])
+
+(defn front-page
+  "Front page info"
+  [e! {user :user :as app}]
+  [:div
+
+   (when test-env?
+     [test-env-warning])
+
+   [:h1 "Hello!"]
+   [:h2 "Hello!"]
+   [:h3 "Hello!"]
+   [:h4 "Hello!"]
+   [:h5 "Hello!"]
+   [:h6 "Hello!"]
+   [:span "Span span span"]
+   [:div "Paavo Pesusieni seikkailee"]
```

Comment:
I assume that The Adventures of Paavo the Sponge will probably be deleted as well before going to production :sponge:

Reply:
Eventually, yes.